src/main/kotlin/by/jprof/telegram/opinions/dao/YoutubeDAO.kt

```diff
@@ -9,13 +9,31 @@ class YoutubeDAO(
         private val whiteListTable: String
 ) {
 
-    suspend fun isInWhiteList(channelId: String) : Boolean {
+    suspend fun isInWhiteList(channelId: String): Boolean {
         val item = dynamoDB.getItem {
             it.tableName(whiteListTable)
-            it.key(mapOf("channelId" to AttributeValue.builder().s(channelId).build()))
+            it.key(channelId.toYoutubeWhiteListItem())
         }.await().item()
 
         return !item.isNullOrEmpty()
     }
 
-}
\ No newline at end of file
+    suspend fun addToWhiteList(channelId: String) {
+        dynamoDB.putItem {
+            it.tableName(whiteListTable)
+            it.item(channelId.toYoutubeWhiteListItem())
+        }.await()
+    }
+
+    fun removeFromWhiteList(channelId: String) {
+        dynamoDB.deleteItem {
+            it.tableName(whiteListTable)
+            it.key(channelId.toYoutubeWhiteListItem())
+        }
```

Comment:
```suggestion
        }.await()
```
Should the `await` be called here?